controllers/fenceagentsremediation_controller_test.go

```diff
@@ -252,6 +312,32 @@ func createRunningPod(containerName, podName, nodeName string) *corev1.Pod {
 	return pod
 }
 
+func createTerminatingPod(containerName, podName, nodeName string) *corev1.Pod {
+	pod := buildPod(containerName, podName, nodeName)
+	now := metav1.Now()
+	pod.ObjectMeta = metav1.ObjectMeta{Name: pod.Name, Namespace: pod.Namespace, DeletionTimestamp: &now}
+	ExpectWithOffset(1, k8sClient.Create(context.Background(), pod)).To(Succeed())
+	return pod
+}
+
+func deleteTerminatingPod(podName string) {
+	pod := &v1.Pod{}
+	pod.Name = podName
+	pod.Namespace = defaultNamespace
+	podKey := client.ObjectKey{
+		Namespace: defaultNamespace,
+		Name:      pod.Name,
+	}
+
+	var grace client.GracePeriodSeconds = 0
+	ExpectWithOffset(1, k8sClient.Delete(context.Background(), pod, grace)).To(Succeed())
+
+	EventuallyWithOffset(1, func() bool {
+		err := k8sClient.Get(context.Background(), podKey, pod)
+		return apierrors.IsNotFound(err)
+	}, 10*time.Second, 100*time.Millisecond).Should(BeTrue())
+}
```

Comment:
Unless I am missing something but the timeout could be shorten and poll interval could be higher for deleting objects. Since it is used in _Eventually_, then it might not make any change for running the unit tests in most cases.

Can you use new consts for these variables or use `timeoutDeletion` and `pollInterval` consts?
(this comment is true for the other two apperances of `10*time.Second`, and `100*time.Millisecond`)

Reply:
Thanks :)